qa/smoke-test-http/src/javaRestTest/java/org/elasticsearch/http/RestClusterInfoActionCancellationIT.java

```diff
@@ -77,7 +77,7 @@ private void runTest(String actionName, String endpoint) throws Exception {
         final Cancellable cancellable = getRestClient().performRequestAsync(request, wrapAsRestResponseListener(future));
 
         assertThat(future.isDone(), equalTo(false));
-        awaitTaskWithPrefix(actionName);
+        awaitTaskWithPrefixOnMaster(actionName);
 
```

Comment:
I tried to verify that the task has been attempted by checking the MANAGEMENT pool queue size or completed tasks. The former would be mostly empty and doesn't pass, and the later is just some large arbitrary number. So asserting >0 or not empty doesn't really help here.

Reply:
@fcofdez As discussed, lets go with this mitigation and leave the logs, and if this doesn't help, I guess we need to somehow rewrite the test in a way. I believe this would still be very helpful since this was already failing rarely.